### PR TITLE
Add cluster tag to the KMS keys

### PR DESCRIPTION
**Pull request by pipo02mix**
Towards https://github.com/giantswarm/aws-operator/issues/662

Add cluster tags to the KMS keys too

### CODE REVIEW

File: service/resource/kmskeyv2/resource.go [hunk 1]
```diff
@@ -3,9 +3,14 @@ package kmskeyv2
 import (
 	"fmt"
 
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/kms"
+	"github.com/giantswarm/apiextensions/pkg/apis/provider/v1alpha1"
```
Comment (fgimenez) [inline service/resource/kmskeyv2/resource.go]: Could you pls put together all the gs imports in a separate block?

File: service/resource/kmskeyv2/spec.go [hunk 2]
```diff
@@ -18,8 +18,9 @@ type Clients struct {
 type KMSClient interface {
 	CreateKey(*kms.CreateKeyInput) (*kms.CreateKeyOutput, error)
 	CreateAlias(*kms.CreateAliasInput) (*kms.CreateAliasOutput, error)
+	DescribeKey(*kms.DescribeKeyInput) (*kms.DescribeKeyOutput, error)
```
Comment (fgimenez) [inline service/resource/kmskeyv2/spec.go]: <3

File: service/resource/kmskeyv2/resource.go (hunk 1 above)
Comment (pipo02mix) [reply service/resource/kmskeyv2/resource.go]: I am not sure what do you mean. I thought we set first built-in go libraries, then external from the project, and finally packages from the same project). I have seen it in others files too, so I am bit confuse

Comment (rossf7) [reply service/resource/kmskeyv2/resource.go]: Yes the convention is the 3rd block has packages from the same project. So this looks correct to me.

Comment (fgimenez) [reply service/resource/kmskeyv2/resource.go]: Indeed, didn't notice the external gs projects and assumed that all the gs imports came from this one, sorry about that :)

**Pull request merged**